src/main/java/Bob.java

```diff
@@ -0,0 +1,119 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Bob {
+    public static void main(String[] args) {
+
+        displayWelcomeMessage();
+        List<Task> list = new ArrayList<>();
+
+        loop: while (true) {
+
+            Scanner in = new Scanner(System.in);
+            String line = in.nextLine();
+
+            String command = line.split(" ")[0];
+            String content;
+
+            switch (command) {
+            case "todo":
+                content = line.split(" ", 2)[1];
+                Task t = new Task(content);
+                list.add(t);
+
+                displayHorizontalLine();
```

Comment:
Agreed with @JustinSoh 

Consider creating a class or methods to handle outputs to user